### ISSUE TITLE
mis-match in documentation and actual installation

### ISSUE THREAD

**Issue by brian69626d** [odpi/egeria#3745]
It seems that the installation guide is no longer up-to-date with the current build: 
https://egeria.odpi.org/open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/task-installing-egeria.html

Egeria is cloned from master and successfully build with maven. But when I was following the installation process, I noticed a few differences in the tutorial and the actual file structure:
1. there is no secondary omag-server* directory within the egeria-install/egeria-omag-2.4-SNAPSHOT/server folder. The jar file is located under server folder itself. 
2. If one try to launch Egeria in egeria-install/egeria-omag-2.4-SNAPSHOT/server folder, a file not found exception would be triggered because java could not locate egeria-install/egeria-omag-2.4-SNAPSHOT/server/truststore.p12. The truststore.p12 file is located in the parent directory of server folder. 
I have tried the following work around: a. copy truststore.p12 into server folder; b. launch the jar file from egeria-install/egeria-omag-2.4-SNAPSHOT folder as follow: java -Dloader.path=server/lib -jar server/server-chassis-spring-2.4-SNAPSHOT.jar. 
Also, it appears that setting strict.ssl=false still tries to fetch truststore.p12.

**Comment by planetf1** [comment]
Thanks. 

Thanks
1. agreed - this should be corrected - feel free to create a PR, or if you prefer we can fix.
2. This is #3705 where there's a little more content. Unfortunately it's partly down to not completely finishing the work for #1403 - it should launch without error in my view, plus we need to ensure the documentation for configuring certs is adequate. The docker image we created will launch the jar without problem. i will try and look at this sooner.....

**Comment by wbittles** [comment]
I've encountered this while automating the setup of an Egeria Integration Daemon configuration and it was costly time ways to resolve it. I realise the work is ongoing but perhaps changing the docs to move the following up to the parent (server) directory
from
`java -Dserver.port=9443 -jar server-chassis-spring-2.6-SNAPSHOT.jar`
to
`java -Dserver.port=9443 -jar ./server/server-chassis-spring-2.6-SNAPSHOT.jar`

could act as an interim fix.

**Comment by planetf1** [comment]
Thanks for the comments.

Fixed the first two points with some minor clarifications in the docs (though I think we need to revisit them more broadly)

The strict.ssl makes a minor change to how certificates are validated in the spring based server.  (for self signed certs). However the code doesn't prevent the .p12 from being loaded - more significant changes would be needed for this.
For now I think it best to concentrate on ensuring it's clear how to start the server/document how to setup the certificates

We did consider some more custom code to setup the correct security context, but it added more complexity, and itself risk for error, vs sticking as close to standard java process behaviour as possible. If we really need to we can revisit.
